Doc/library/shlex.rst

```diff
@@ -43,15 +43,16 @@ The :mod:`shlex` module defines the following functions:
    string that can safely be used as one token in a shell command line, for
    cases where you cannot use a list.
 
-   This idiom would be unsafe::
+   This idiom would be unsafe:
```

Comment:
Hi @berkerpeksag , to make the test after _The quoting is compatible with UNIX shells_ to pass, `remote_command` should be defined.  In order for `remote_command` to be defined, we need to execute this test and the next one, that is why I changed `::`` to ``:`.  If you want to use ``::`` instead of `:`, than you have to skip the test that uses the `remote_command` name, but I do not see any good reason for that.